Classes/Settings/SettingsViewController.swift

```diff
@@ -115,6 +118,12 @@ NewIssueTableViewControllerDelegate {
 		presentSafari(url: url)
     }
 
+    func onViewBlog() {
+        guard let url = URL(string: "http://blog.githawk.com/")
```

Comment:
This is less a note with this PR but more a general statement, are we okay with generic strings like this being placed throughout the codebase? Or would we rather move them to either a constants file, or maybe the Info.plist under it's own dictionary? Easier to manage long-term?

Reply:
This the best shot would be to store them in a constants file. And then call it by: `Constants.url.blog` or something similar

Reply:
We already have [Strings.swift](https://github.com/rnystrom/GitHawk/blob/master/Classes/Views/Strings.swift), maybe that works for now as well?

Reply:
Created a new issue for this:

https://github.com/rnystrom/GitHawk/issues/498

Reply:
Awesome!